spec/forms/candidate_interface/degree_year_form_spec.rb

```diff
@@ -21,11 +21,39 @@
         )
       end
     end
+
+    it 'is invalid if the degree is incomplete and the award year is in the past' do
+      Timecop.freeze(Time.zone.local(2008, 1, 1)) do
+        degree = build(
+          :degree_qualification,
+          qualification_type: 'BSc',
+          predicted_grade: true,
+          start_year: '2008',
+          award_year: '1990',
+        )
+
+        degree_form = described_class.new(degree: degree, award_year: '1990')
```

Comment:
Would be interested to know if there's a better approach... having to pass in `degree` as the validation looks for the presence of `predicted_grade`, so without it it returns:
```
undefined method `predicted_grade' for nil:NilClass
```

Reply:
Will the future validation date (2023 in this case) always update itself? Eg, when candidates apply in 2022, will the message say 'Enter a year before 2023'?)

I'm also wondering if we should split this into 2 questions if someone hasn't completed their degree yet, eg:
What did you start your degree?
When will you graduate?

Because it doesn't make sense for the question to be in the past tense if someone hasn't completed their degree yet.

![Screenshot 2021-07-26 at 09 32 44](https://user-images.githubusercontent.com/85290597/126959228-a983a63c-a9c7-410c-9975-f586aec3b308.png)

Reply:
@B-Bonner Just to confirm – what you've included here is actually an existing validation on the form (which will automatically update the value).

To trigger my validation you need to select that you haven't completed the degree on the previous page:
![image](https://user-images.githubusercontent.com/47917431/126970127-a28ec086-6bde-4a8d-aaec-c0371e88f460.png)

I'm happy to discuss changing the copy though!